.github/workflows/check_node_latest.yml

```
@@ -0,0 +1,38 @@
+name: Check Reflex run on Node.js Latest
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+env:
+    TELEMETRY_ENABLED: false
+
+jobs:
+    check_latest_node:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: ['3.12']
+                node-version: ['node']
+        steps:
+            - uses: actions/checkout@v4
+            - uses: ./.github/actions/setup_build_env
+              with:
+                python-version: ${{ matrix.python-version }}
+                run-poetry-install: true
+                create-venv-at-path: .venv
+            - uses: actions/setup-node@v4
+              with:
+                node-version: ${{ matrix.node-version }}
+            - run: |
+                poetry run uv pip install playwright pytest-playwright pyvirtualdisplay pillow
+                poetry run playwright install --with-deps
+            - run: |
+                poetry run pytest integration/test_node_version.py
```

```suggestion
                poetry run pytest integration/
```

can we run _all_ of the integration tests with latest node to improve our coverage and catch more potential errors?